projects/core/src/lib/download-vector-layer/download-vector-layer.component.html

```diff
@@ -0,0 +1,71 @@
+<geoengine-sidenav-header>Download Vector Layer</geoengine-sidenav-header>
```

Comment:
Download vector layer looks exactly the same as download raster layer.

Isn't it possible to use common code instead of duplicating it?

Reply:
https://github.com/geo-engine/geoengine-ui/pull/722/commits/937713ca40b7a44c1e1bae00a77645322c63bc33